pkg/image/export/main.go

```diff
@@ -78,7 +67,7 @@ func run(systemChartPath, chartPath string, imagesFromArgs []string) error {
 	if err != nil {
 		return err
 	}
-
+	rancherVersion := settings.GetRancherVersion()
```

Comment:
The logic here is different, `settings.GetRancherVersion()` is not following same logic than the removed code, https://github.com/rancher/rancher/pull/33576/files#diff-6677f48204c790d8027374cbc1843b533171441b0c777b611b264cf3789ffbf2L56-L67, is this intended?? Not using TAG env var anymore??

Reply:
My understanding is that this code is from before we had the GetRancherVersion function we use everywhere else, but essentially does the same thing with the exception of needing to set a TAG env var instead of reading the settings, and the case where the version has the suffix "-head" which GetRancherVersion will change into "2.6.99" and the removed logic won't.